### ISSUE TITLE
Setup wiki

### ISSUE THREAD

**Issue by JoshHiles** [scientistproject/Scientist.net#140]
## Description

**As a** member of the community,
**I want** a wiki 
**So that** we can read examples, setup & other information to users

## Acceptance Criteria

- Installation instructions
- Designing
- Usage
  - Comparison
  - Context
  - Expensive setup
  - Cleaning up
  - Ignoring Mismatches
  - Enabling / Disabling experiments
  - Experiment ramp up
  - Asynchronous usage
  - Testing
  - Handling errors
  - Ignoring results
  - Multiple experiments
- Results
- Finishing
- Contributing
  - Installation
  - Testing
  - PR 
- Anything else....

**Comment by marblekirby** [comment]
I could kick this off on the core-rewrite branch but dont have permission to access the current / create wiki pages.

**Comment by M-Zuber** [comment]
Do we want a wiki or GH-Pages?

**Comment by JoshHiles** [comment]
It's a good point, i think the wiki can only be mainained by the contributers but the GitHub pages can be publicly maintained, only caveat is that you have to create another repo for the GitHub pages

**Comment by M-Zuber** [comment]
They no longer have support for branch based sites?

**Comment by JoshHiles** [comment]
I'm wrong! https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site we can have a branch for the pages

**Comment by marblekirby** [comment]
i think we can have a wiki on this github. Should we go with that?

**Comment by JoshHiles** [comment]
We do have a wiki on this github, it would be nice to add you @marblekirby to the contributors team, what do we think @scientistproject/scientist-net-maintainers

**Comment by paulbreen-symphonytalent** [comment]
I agree @JoshHiles

**Comment by JoshHiles** [comment]
I think @shiftkey is the only person able to add people to the contributors team

**Comment by shiftkey** [comment]
I've promoted @JoshHiles and @M-Zuber to maintainers of that team - that should allow you to add more contributors as needed

**Comment by JoshHiles** [comment]
Sorry @shiftkey we will need privileges on the scientist project organisation as well to add team members

**Comment by shiftkey** [comment]
@JoshHiles apologies - you and @M-Zuber are now organization owners

**Comment by marblekirby** [comment]
Thanks for inviting me to the organisation. Can someone post the link to the existing wiki?

**Comment by M-Zuber** [comment]
It would be here https://github.com/scientistproject/Scientist.net/wiki
I do not see any actual content though, so maybe the first step is to get the contents of the Readme into the wiki

**Comment by marblekirby** [comment]
Yes, i found that wiki, with no content, so assumed that wasnt the right one! 
I can start populating that with the readme into wiki files.
